ykrt/src/mt.rs

```diff
@@ -1039,20 +1143,25 @@ mod tests {
     }
 
     fn expect_start_tracing(mt: &Arc<MT>, loc: &Location) {
-        let TransitionControlPoint::StartTracing(hl) = mt.transition_control_point(loc) else {
+        let TransitionControlPoint::StartTracing(hl) =
+            mt.transition_control_point(loc, 0 as *mut c_void)
+        else {
             panic!()
         };
         MTThread::with(|mtt| {
             *mtt.tstate.borrow_mut() = MTThreadState::Tracing {
                 hl,
                 thread_tracer: Box::new(DummyTraceRecorder),
                 promotions: Vec::new(),
+                frameaddr: 0 as *mut c_void,
             };
         });
     }
 
     fn expect_stop_tracing(mt: &Arc<MT>, loc: &Location) {
-        let TransitionControlPoint::StopTracing = mt.transition_control_point(loc) else {
+        let TransitionControlPoint::StopTracing =
+            mt.transition_control_point(loc, 0 as *mut c_void)
```

Comment:
Clippy will probably say to use `ptr::null_mut()`.

Reply:
There are no warnings from Clippy. Note this is a `c_void`.

Reply:
It should work since `null_mut<T>()` parametrised. Fewer magic numbers, right?